### DISCUSSION TITLE
iOS App how to search in note?

### DISCUSSION THREAD

**Discussion by r2r72** [glushchenko/fsnotes#1306] [Q&A] [answered]
Hi, 

search button allow me to find a note, but if I need to find something inside how to do it? Is there is any way how to do it? On Mac standard Cmd+F works perfectly for me, sometimes need the same in iOS.

Best regards
Artur

**Comment by glushchenko** [comment, marked as answer]
Hello! In FSNotes for iOS, only global search is implemented, after switching to a note, the searched text is highlighted.

**Reply by r2r72** [reply]
Thank you!
